pipelines/sklearn.linear_model.LogisticRegression.json

```diff
@@ -0,0 +1,13 @@
+{
+	"metadata": {
+		"name": "sklearn.linear_model.LogisticRegression",
+		"data_type": "single_table",
+		"task_type": "classification",
+	},
+	"validation": {
+		"dataset": "boston"
```

Comment:
Since this is a classification, the dataset used should be `iris` instead of `boston`